python/hail/tests/tests.py

```diff
@@ -329,7 +316,8 @@ def test_dataset(self):
 
         sample2.split_multi().impute_sex().variant_schema
 
-        self.assertTrue(isinstance(sample2.genotype_schema, TGenotype))
+        # FIXME htsGenotypeSchema
```

Comment:
can you explain this a bit?

Reply:
Yeah, I wanted to check the genotype_schema is hts_genotype_schema, but I need to export it to python.  Will do.